src/validator.js

```diff
@@ -0,0 +1,39 @@
+import { MAX_VALUE } from './constants/calculatorRule';
+import { ERROR_MESSAGE } from './constants/message';
+import { OPERATIONS } from './constants/operations';
+
+class Validator {
+  static validate(first, second, operation) {
+    this.isInvalidFormula(first, second, operation);
+    this.isInvalidOperand(first, second);
+    this.isInvalidOperation(operation);
+    this.isOverDigit(first, second);
+  }
+
+  static isInvalidFormula(first, second, operation) {
+    if (['', undefined].includes(first) || ['', undefined].includes(second) || ['', undefined].includes(operation)) {
```

Comment:
각 param 값들이 유효한지 아닌지를 확인하는 조건문이네요.

```suggestion
    if (!first || !second || !operation) {
```

이렇게 간결하게도 표현할 수 있습니다.